src/ai/backend/client/func/service.py

```diff
@@ -89,6 +89,12 @@ async def create(
         image: str,
         model_id_or_name: str,
         initial_session_count: int,
+        /,
+        resources: Mapping[str, str | int],
+        resource_opts: Mapping[str, str | int],
+        domain_name: str,
+        group_name: str,
+        scaling_group: str,
```

Comment:
/ is required?

Reply:
Not mandatory.

Reply:
Then we just change the position and take it as a keyword argument as before.